redash/handlers/static.py

```diff
@@ -13,7 +14,7 @@ def render_index():
         response = render_template("multi_org.html", base_href=base_href())
     else:
         full_path = safe_join(settings.STATIC_ASSETS_PATH, "index.html")
-        response = send_file(full_path, **dict(cache_timeout=0, conditional=True))
+        response = send_file(full_path, **dict(max_age=0, conditional=True))
```

Comment:
Changed in version 2.0: max_age replaces the cache_timeout parameter. conditional is enabled and max_age is not set by default.

https://flask.palletsprojects.com/en/2.3.x/api/#flask.send_file